src/createStore.test.ts

```diff
@@ -0,0 +1,54 @@
+import { compressToUTF16 } from 'lz-string';
+import { createStore } from './createStore';
+import { ISavedState, loadStoredState } from './localStorage';
+import { defaultState, IState } from './reducers/states';
+import { history } from './utils/history';
+
+const mockChangedStoredState: ISavedState = {
+  session: {
+    accessToken: 'yep',
+    refreshToken: 'refresherOrb',
+    role: undefined,
+    name: 'Jeff'
+  },
+  playgroundEditorValue: 'Nihao everybody',
+  playgroundIsEditorAutorun: true
+};
+
+const mockChangedState: IState = {
+  ...defaultState,
+  session: {
+    ...defaultState.session,
+    accessToken: 'yep',
+    refreshToken: 'refresherOrb',
+    role: undefined,
+    name: 'Jeff'
+  },
+  workspaces: {
+    ...defaultState.workspaces,
+    playground: {
+      ...defaultState.workspaces.playground,
+      editorValue: 'Nihao everybody',
+      isEditorAutorun: true
+    }
+  }
+};
+
+describe('CREATESTORE', () => {
```

Comment:
You dont have to capitalize everything! You can do `createStore() function` for e.g. (same for the other describes)